### DISCUSSION TITLE
Package not running

### DISCUSSION THREAD

**Discussion by masteling** [SciFracX/FractionalCalculus.jl#8] [Q&A] [answered]
I am trying to use this package but keep getting errors:
![image](https://user-images.githubusercontent.com/24229030/182229130-a44fd72f-5571-428c-af41-bec2cb3d6b5e.png)

I have tried reversing to earlier commits but haven't been able to find a working version.
Can you point me to the latest working version?

Thank you

**Comment by ErikQQY** [comment, marked as answer]
Hello there! I think that error is caused by the MittagLeffler.jl package. You may need to remove the MittagLeffler.jl. Then you could install the latest version of FractionalCalculus.jl@0.2.10

Besides, there are built in Mittag Leffler functions in FractionalCalculus.jl, which is more efficient than the implementation in MittagLeffler.jl:

![EA3684D6-A1AB-4B11-B1C8-90B4F75E6ED3](https://user-images.githubusercontent.com/52615090/182309847-4be3349b-87d2-4cd3-a063-590a66183120.png)

After using the FractionalCalculus.jl, you can call the Mittag Leffler function by:

```julia
julia> mittleff(alpha, z)
julia> mittleff(alpha, beta, z)
julia> mittleff(alpha, beta, gamma, z)
```

For more details about Mittag Leffler function, please see the documentation: https://scifracx.org/FractionalDiffEq.jl/dev/mittagleffler/
